AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/MutationSync/OutgoingMutationQueue/ProcessMutationErrorFromCloudOperation.swift

```diff
@@ -12,18 +12,28 @@ import AWSPluginsCore
 
 /// Checks the GraphQL error response for specific error scenarios related to data synchronziation to the local store.
 /// 1. When there is a "conditional request failed" error, then emit to the Hub a 'conditionalSaveFailed' event.
+/// 2. When there is a "conflict unahandled" error, trigger the conflict handler and reconcile the state of the system.
 @available(iOS 13.0, *)
-class ProcessMutationErrorFromCloudOperation: Operation {
+class ProcessMutationErrorFromCloudOperation: AsynchronousOperation {
 
+    private let dataStoreConfiguration: DataStoreConfiguration
+    private let storageAdapter: StorageEngineAdapter
     private let mutationEvent: MutationEvent
     private let error: GraphQLResponseError<MutationSync<AnyModel>>
```

Comment:
We should rename this 'error' because we have local variables in functions that also go by the name of 'error' as well.

Reply:
renamed to type name `graphQLResponseError`